nats-streaming-http-connector/test/consumer/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/nats-io/nats.go"
+	"github.com/nats-io/stan.go"
+)
+
+func main() {
+	nc, err := nats.Connect("nats://nats:4222")
+	if err != nil {
+		log.Fatal(err)
+	}
+	sc, err := stan.Connect("my-stan", "stan-sub", stan.NatsConn(nc))
+	if err != nil {
+		log.Fatal(err)
+	}
+	_, err = sc.Subscribe("response-topic", func(m *stan.Msg) {
+		msg := string(m.Data)
+		fmt.Println(msg)
+	}, stan.DeliverAllAvailable())
+	fmt.Println("All messages consumed")
```

Comment:
Test logs look like this 

```
All messages consumed
Test1
....
Test99
```

All messages consumed should be at last?

Reply:
Also as I see in producer code
```
for i := 0; i < 100; i++ {
		sc.Publish("request-topic", []byte("Test"+strconv.Itoa(i)))
	}
```
Please check if above message seq is as desired.